installation/cmd/run.sh

```diff
@@ -113,6 +125,21 @@ function mount_minikube_ca_to_oathkeeper() {
  -p '{"spec":{"template":{"spec":{"containers":[{"name": "'$OATHKEEPER_CONTAINER_NAME'","volumeMounts": [{ "mountPath": "'/etc/ssl/certs/mk-ca.crt'","name": "minikube-ca-volume","subPath": "mk-ca.crt"}]}]}}}}'
 }
 
+function set_oidc_config() {
```

Comment:
As far as I understand, the `.global.cockpit.auth.*` values will be used in the in-cluster compass-console. Wouldn't it be a good idea to mention that we should set the respective flags when executing the installation script [here](https://github.com/kyma-incubator/compass/blob/main/docs/compass/04-01-installation.md#local-minikube-installation)?

Reply:
Absolutely! I'd forgotten that.